reference/configuration/framework.rst

```diff
@@ -55,7 +55,7 @@ Configuration
 * `session`_
     * `storage_id`_
     * `handler_id`_
-    * `name`_
+    * :ref:`name <session-name>`
```

Comment:
@javiereguiluz Might cause conflicts as these labels are global and not limited to the current document.